README.md

```diff
@@ -64,9 +64,18 @@ end
 TODO
 ```
 
-## Documentation
+## Documentation - APIS
 
-TODO (e.g. link to wiki page)
+
+Following table shows the API endpoints of the Policy Manager.
+
+| Action | HTTP Method | Endpoint |
+| --------------- | ------- | -------------------------------------------- |
+| Create a Policy | `PUT` |`/api/policymngr/v1/policy-descriptor`|
```

Comment:
@jbonnet  I used PUT because according to http://restcookbook.com/HTTP%20Methods/put-vs-post/ , PUT can be used for updating a resource completely through a specific resource. In our case at both add and edit of a policy, the descriptor is totally added or replaced at the catalogs. Since PUT and POST are too close as concepts, we can switch to POST without problem.